tests/graphql/operations/jsonplaceholder.graphql

```diff
@@ -0,0 +1,46 @@
+#> server-sdl
+schema
+  @server(port: 8000, graphiql: true, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true) {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+  user(id: Int!): User @http(path: "/users/{{args.id}}")
+}
+
+type User {
+  id: Int!
+  name: String!
+  username: String!
+  email: String!
+  phone: String
+  website: String
+}
+
+type Post {
+  id: Int!
+  userId: Int!
+  title: String!
+  body: String!
+  user: User @http(path: "/users/{{value.userId}}")
+}
+
+#> client-query
+query @diagnostic(count: 0) {
+  user(id: 1) {
+    id
+    name
+    username
+  }
+}
+
+#> client-query
+query @diagnostic(count: 1) {
+  user(id: 1) {
+    id
+    nome
+    username
+  }
+}
```

Comment:
Tests in `/graphql` and `/http` exist because we have written that logical components that execute those bits. However here we are leveraging what async_graphql provides us. We should just test that when we call the methods on async_graphql, it returns results in the format we expect and that should be enough. 

The tests written in `operation.rs` should be enough for this usecase.